README.md

```diff
@@ -28,7 +37,9 @@ This repository contains the source code for the GyanaGuru website, an online le
 # Getting Started
 ## How To SetUp
 - Clone This Project
```

Comment:
Rather than cloning the project....

it would be great if you could add these following steps

- Fork The Repo
` ` `  adding necessary steps(eg fork button)` ` `

- Clone your forked repo
` ` ` Adding necessary steps (eg git clone etc)` ` `